src/commands.rs

```diff
@@ -33,6 +33,23 @@ impl Commands {
         &mut self,
         command: &'static str,
         handler: impl Fn(Args) -> Result<()> + Send + Sync + 'static,
+    ) {
+        self.__add(command, handler, false)
+    }
+
+    pub(crate) fn add_hidden(
+        &mut self,
+        command: &'static str,
+        handler: impl Fn(Args) -> Result<()> + Send + Sync + 'static,
+    ) {
+        self.__add(command, handler, true)
+    }
+
+    fn __add(
+        &mut self,
+        command: &'static str,
+        handler: impl Fn(Args) -> Result<()> + Send + Sync + 'static,
+        hidden: bool,
```

Comment:
Whats the motivation here?

Reply:
I felt like i was adding lots of commands that were error cases / variants on the same command and didn't want to clutter up the help menu

Reply:
Im ok with this for now.  Its kinda hacky and I dont really like it but its a good sign that we need to do better when generating the help menu.